components/Pagination.js

```diff
@@ -0,0 +1,26 @@
+import styles from "../styles/Home.module.css";
+const Pagination = ({ items, pageSize, currentPage, onPageChange }) => {
+  const pagesCount = Math.ceil(items / pageSize);
+
+  if (pagesCount === 1) return null;
+  const pages = Array.from({ length: pagesCount }, (_, i) => i + 1);
+
+  return (
+    <div>
+      <ul className={styles.pagination}>
+        {pages.map((page) => (
+          <li
+            key={page}
+            className={
```

Comment:
You can use classNames package. It is a simple JavaScript utility for conditionally joining classNames together.
https://www.npmjs.com/package/classnames

Reply:
Thanks, I will look it up!)